packages/mui-styles/package.json

```diff
@@ -55,14 +55,14 @@
     "clsx": "^1.1.1",
     "csstype": "^3.0.10",
     "hoist-non-react-statics": "^3.3.2",
-    "jss": "^10.8.2",
-    "jss-plugin-camel-case": "^10.8.2",
-    "jss-plugin-default-unit": "^10.8.2",
-    "jss-plugin-global": "^10.8.2",
-    "jss-plugin-nested": "^10.8.2",
-    "jss-plugin-props-sort": "^10.8.2",
-    "jss-plugin-rule-value-function": "^10.8.2",
-    "jss-plugin-vendor-prefixer": "^10.8.2",
+    "jss": "^10.9.0",
```

Comment:
@oliviertassinari how do we handle bumping of dependencies in a minor? Is this safe to be merged?

Reply:
On the other hand, this is a dependency, for some reason I thought is a peer dependency, but still I am not sure if this would not be a breaking change.